src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt

```diff
@@ -209,7 +233,6 @@ class SuggestionDescriptionPanelFromLS(
 
     private fun getCustomScript(): String {
         return """
-            (function () {
```

Comment:
When extracting the script, we need acces to the variables which are locked in the scope of this self invoked function and can't be accessed outside, we can have the logic encapsulated in a self invoked function in the LS, that would require some minimal changes to all IDE's that inject scripts.